PostHogExample/AppDelegate.swift

```diff
@@ -17,18 +17,22 @@ class AppDelegate: NSObject, UIApplicationDelegate {
         // the ScreenViews for SwiftUI does not work, the names are not useful
         config.captureScreenViews = false
         config.captureApplicationLifecycleEvents = false
-        config.flushAt = 1
-        config.flushIntervalSeconds = 30
+//        config.flushAt = 1
+//        config.flushIntervalSeconds = 30
         config.debug = true
         config.sendFeatureFlagEvent = false
         config.sessionReplay = true
         config.sessionReplayConfig.screenshotMode = true
+        config.sessionReplayConfig.maskAllTextInputs = true
+        config.sessionReplayConfig.maskAllImages = true
 
         PostHogSDK.shared.setup(config)
-        PostHogSDK.shared.debug()
+//        PostHogSDK.shared.debug()
```

Comment:
Do we want to add these back?

Reply:
Nope because `config.debug = true`, so it was just repeating.